features/guest.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Guest order Registration
+
+Scenario: Guest order placement using (Negative scenario)
```

Comment:
@karthikeyan7010 Again irrelevant scenarios. No background and doesn't make sense.